test/e2e/bandwidth_test.go

```diff
@@ -26,17 +26,50 @@ import (
 
 const iperfPort = 5201
 
-// TestBenchmarkBandwidthIntraNode runs the bandwidth benchmark between Pods on same node.
-func TestBenchmarkBandwidthIntraNode(t *testing.T) {
-	skipIfNotBenchmarkTest(t)
-	skipIfNotIPv4Cluster(t)
+// TestBandwidth is the top-level test which contains all subtests for
+// Bandwidth related test cases so they can share setup, teardown.
+func TestBandwidth(t *testing.T) {
 	skipIfHasWindowsNodes(t)
+	skipIfNotRequired(t, "mode-irrelevant-case")
 
 	data, err := setupTest(t)
 	if err != nil {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
+
+	t.Run("testPodTrafficShaping", func(t *testing.T) { testPodTrafficShaping(t, data) })
+}
+
+// TestAllBenchmarkBandwidth is the top-level benchmark test which contains all subtests for
+// Bandwidth related test cases so they can share setup, teardown.
+func TestAllBenchmarkBandwidth(t *testing.T) {
+	skipIfNotBenchmarkTest(t)
+	skipIfHasWindowsNodes(t)
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	t.Run("TestAllBenchmarkBandwidth", func(t *testing.T) {
```

Comment:
Same comment as previous one: "All" in the test name, unnecessary test group

Reply:
done